ocaml/xapi/message_forwarding.ml

```diff
@@ -698,10 +698,10 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
         let unfinished = List.exists (fun state -> state = `pending) statuses in
         if unfinished
         then begin
-          let from = Helpers.call_api_functions ~__context
-              (fun rpc session_id -> Client.Event.from ~rpc ~session_id ~classes ~token ~timeout:30.0) in
+          let timeout = 30.0 in
+          let classes = classes in
```

Comment:
Really?

Reply:
To be extra sure...
oops, there was a point where some of that line was different and thus useful. Removed now